HousingSearchListener/V1/Boundary/EventTypes.cs

```diff
@@ -16,5 +16,8 @@ public static class EventTypes
 
         public const string AssetCreatedEvent = "AssetCreatedEvent";
         public const string AssetUpdatedEvent = "AssetUpdatedEvent";
+
+        public const string ProcessStartedEvent = "ProcessStartedEvent";
+        public const string ProcessUpdatedEvent = "ProcessUpdatedEvent";
```

Comment:
Would we also need the process closed/cancelled events here?

Reply:
Is that part of a separate ticket? If yes, happy to approve this PR

Reply:
No, it's not. I'd say we need to make another PR and then deploy all to staging after it's done?

Reply:
We've created a new [subticket](https://hackney.atlassian.net/browse/MTTL-3319) for ProcessClosed, ProcessCompleted and ProcessCancelled events so that this once can be deployed to staging.